src/dc_tools.rs

```diff
@@ -24,6 +25,31 @@ pub fn dc_exactly_one_bit_set(v: libc::c_int) -> bool {
     0 != v && 0 == v & (v - 1)
 }
 
+pub struct CListIterator {
```

Comment:
could you PR this to https://github.com/filecoin-project/rust-fil-proofs/pull/836 directly, and implement `IntoIter` for `clist` there? (that code only exists for use in here)

Reply:
`clist` is defined in `mmime=0.1.1`. `clist` is not even mentioned in project you linked (at least, at master). Again, what do you want me to do?

Reply:
I am so sorry, I pasted the entirely wrong link -.-

https://github.com/dignifiedquire/mmime/ is the link I meant to paste